src/core/nconn/nconn_tls.cc

```diff
@@ -1186,6 +1238,34 @@ int32_t nconn_tls::ncconnect()
                                 return NC_STATUS_ERROR;
                         }
                 }
+        
+        // David S - KTLS debug
+        // The m_tls_bio was kept for convenience, but it seems to be more robust
+        // to rather make use of the SSL_get_wbio/rbio functions when checking for KTLS
+        // since calls to SSL_set_fd exist elsewhere in the hurl source code which could 
+        // potentially 'free' the BIOs assigned to m_tls and thereafter create new BIOs
+        // and attach these to m_tls.
+        //if (BIO_get_ktls_send(m_tls_bio))
+        if (BIO_get_ktls_send(SSL_get_wbio(m_tls)))
```

Comment:
This was a slip from my side. I used this code mainly for debugging purposes while doing the KTLS modifications.

I think it might be worth cleaning this up and making it part of the verbose mode output? This should make it simple for a user to verify if hurl is actually using KTLS or not.

Reply:
Moved to verbose mode output.